README.rst

```diff
@@ -107,6 +107,57 @@ And if you'd prefer to see ``assert`` being used, all ``verify`` assertions will
     assert Truthy(1)
     assert expect(1, Truthy(), Number())
 
+More natural syntax
+===================
+
+You can also use more natural syntax using ``ensure`` alias for ``expect`` and
+prefixes ``to_be_*`` or ``is_*``:
+
+.. code-block:: python
+
+    expect(some_var).to_be_int().to_be_less_or_equal(5).to_be_not_list()
+    ensure(some_var).is_int().is_less_or_equal(5).is_not_list()
+    # Both above lines are the same as:
+    expect(some_var).Int().LessOrEqual(5).NotList()
+
+All assertions can be also used as lowercased methods:
```

Comment:
Instead of `lowercased methods`, either `underscore case` or `snake case` (https://en.wikipedia.org/wiki/Snake_case).